AmplifyPlugins/Auth/AWSCognitoAuthPlugin/Dependency/AuthenticationProviderAdapter+DeleteUser.swift

```diff
@@ -0,0 +1,42 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Amplify
+#if COCOAPODS
+import AWSMobileClient
+#else
+import AWSMobileClientXCF
+#endif
+
+extension AuthenticationProviderAdapter {
+
+    func deleteUser(request: AuthDeleteUserRequest, completionHandler: @escaping (Result<Void, AuthError>) -> Void) {
+        // By default, ASWMobileClient calls signOut internally during deleteUser.
+        // For Amplify, we instead call Amplify's signOut function, which contains some higher level logic.
```

Comment:
Remove this comment? Amplify is not calling signOut right?

Reply:
done